models/shopify__daily_shop.sql

```diff
@@ -0,0 +1,213 @@
+with shop as (
+
+    select *
+    from {{ var('shopify_shop') }}
+),
+
+calendar as (
+
+    select *
+    from {{ ref('shopify__calendar') }}
+    where cast({{ dbt.date_trunc('day','date_day') }} as date) = date_day
+
+),
+
+shop_calendar as (
+
+    select
+        calendar.date_day,
+        shop.*
+
+    from calendar
+    join shop on
+        cast(shop.created_at as date) <= calendar.date_day
```

Comment:
Formatting suggestion
```suggestion
   join shop 
        on cast(shop.created_at as date) <= calendar.date_day
```

Additionally should this be a left join?